src/main/kotlin/Main.kt

```diff
@@ -1,5 +1,5 @@
 import java.util.Scanner
-
+// хуй
```

Comment:
напиши с большой буквы, пожалуйста
```suggestion
// Хуй
```

Reply:
Пошел в жопу